release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -42,6 +42,40 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Language
 
+##### Support for Module level complex variables
+
+Variable declarations with list, mapping, and error binding patterns are now allowed at module level. Unlike simple variables, these variables must be initialized in the declaration.
+
+Complex variable declarations cannot contain `isolated` or `configurable` qualifier.
+
+```ballerina
+// Module-level variable declaration with a list binding pattern. 
+[int, float] [a, b] = [1, 2.5];
+
+// Module-level variable declaration with a mapping binding pattern.
+Person {name: firstname, married: isMarried} = {name: "Jhone", married: true};
+
+type Person record {|
+    string name;
+    boolean married;
+|};
+
+// Module-level variable declaration with an error binding pattern.
+error error(message) = error("error message");
```

Comment:
For the expressions on the RHS, shall we use function calls like you had done for error earlier? It may not be obvious to users the advantages of binding patterns when we create a new value just for this.

For the error binding pattern, let's also bind a detail field.